content/blog/jamstack-with-pulumi/index.md

```diff
@@ -0,0 +1,359 @@
+        'forwardedValues': {
+            'cookies': { 'forward': 'none' },
+            'queryString': False,
+        },
+        'minTtl': 0,
+        'defaultTtl': TEN_MINUTES,
+        'maxTtl': TEN_MINUTES,
+    },
+    # PriceClass_100 is the lowest cost tier (US/EU only).
+    price_class= 'PriceClass_100',
+    custom_error_responses=[{
+            'errorCode': 404,
+            'responseCode': 404,
+            'responsePagePath': '/404.html'
+        }],
+    # Use the certificate we generated for this distribution.
+    viewer_certificate={
+        'acmCertificateArn': certificate_arn,
+        'sslSupportMethod': 'sni-only',
+    },
+    restrictions={
+        'geoRestriction': {
+            'restrictionType': 'none'
+        }
+    },
+    # Put access logs in the log bucket we created earlier.
+    logging_config={
+        'bucket': logs_bucket.bucket_domain_name,
+        'includeCookies': False,
+        'prefix': f'${target_domain}/',
+    },
+    # CloudFront typically takes 15 minutes to fully deploy a new distribution.
+    # Skip waiting for that to complete.
+    wait_for_deployment=False)
+```
+
+To complete the deployment, we set the `alias_a_record` to point the CDN to our domain name in Route 53.
+
+```python
+def create_alias_record(target_domain, distribution):
+    """
+    Create a Route 53 Alias A record from the target domain name to the CloudFront distribution.
+    """
+    subdomain, parent_domain = get_domain_and_subdomain(target_domain)
+    hzid = pulumi_aws.route53.get_zone(name=parent_domain).id
+    return pulumi_aws.route53.Record(target_domain,
+        name=subdomain,
+        zone_id=hzid,
+        type='A',
+        aliases=[
+            {
+                'name': distribution.domain_name,
+                'zoneId': distribution.hosted_zone_id,
+                'evaluateTargetHealth': True
+            }
+        ]
+    )
+
+alias_a_record = create_alias_record(target_domain, cdn)
+```
+
+In a single Python script, we've created all the cloud resources to deploy our content and served it via a CDN.
+
+## Conclusion
+
+Building modern websites is evolving from provider-managed Content Management Systems or user-managed [LAMP](https://en.wikipedia.org/wiki/LAMP_(software_bundle)) stacks. Static websites deployed in the cloud are popular because they separate content generation from the infrastructure that serves content. However, building and maintaining cloud infrastructure can be complicated and require multiple steps when using web client for configuration. A website using inexpensive storage such as S3 and serving content via Cloudfront can be deployed, versioned, and maintained using your favorite programming language. The Pulumi [examples repository](https://github.com/pulumi/examples) has many more code samples that demonstrate how to build and deploy cloud infrastructure across different cloud providers, using your favorite programming language.
```

Comment:
Would it also be worth adding a sentence that explicitly mentions "for the same example in TypeScript, see https://github.com/pulumi/examples/tree/master/aws-ts-static-website"?

Reply:
Added in the TypeScript examples with language chooser